### PR TITLE
Modernize embulk-filter-rename with v0.11-ready API/SPI

### PR DESCRIPTION
**Pull request by dmikurube**

### CODE REVIEW

File: embulk-filter-rename/src/main/java/org/embulk/filter/rename/RenameFilterPlugin.java [hunk 1]
```diff
@@ -441,12 +469,27 @@ private Schema applyUniqueNumberSuffixRule(Schema inputSchema, UniqueNumberSuffi
...
     }
 
-    private static final ImmutableMap<String, String> CHARACTER_TYPE_KEYWORDS =
-            new ImmutableMap.Builder<String, String>()
-                    .put("a-z", "a-z")
-                    .put("A-Z", "A-Z")
-                    .put("0-9", "0-9")
-                    .build();
+    private static String repeatStrings(final String string, final int count) {
+        final StringBuffer buffer = new StringBuffer();
```
Comment (huylenq) [inline embulk-filter-rename/src/main/java/org/embulk/filter/rename/RenameFilterPlugin.java:473]: Nit: StringBuilder would suffice?

**Review by huylenq — APPROVED** [review]
LGTM

File: embulk-filter-rename/src/main/java/org/embulk/filter/rename/RenameFilterPlugin.java (hunk 1 above)
Comment (dmikurube) [reply embulk-filter-rename/src/main/java/org/embulk/filter/rename/RenameFilterPlugin.java:473]: Umm, I don't remember why I used `StringBuffer`, but `StringBuilder` could be better. I'll have a revisit here later. Thanks!

**Comment by dmikurube** [comment]
Thanks!

**Pull request merged**